extensions/common/azure/azure-blob-core/src/main/java/org/eclipse/edc/azure/blob/api/BlobStoreApi.java

```diff
@@ -44,4 +44,6 @@ public interface BlobStoreApi {
     BlobAdapter getBlobAdapter(String accountName, String containerName, String blobName, String sharedKey);
 
     BlobAdapter getBlobAdapter(String accountName, String containerName, String blobName, AzureSasCredential credential);
+
+    BlobAdapter getBlobAdapter(String accountName, String containerName, String blobName);
```

Comment:
some documentation would be nice. I know, the rest of the interface isn't documented either, but maybe you could put a comment on this added method (e.g. why there is no auth parameter)

Reply:
Sure, no problem. I documented the three getBlobAdapter methods to illustrate the differences. I would've documented the others as well, but as discussed with @bscholtes1A, I think a refactoring would make sense here and I did not want to spent unnecessary efforts. Hope that's fine.